src/components/Filters/index.js

```diff
@@ -261,14 +261,21 @@ class Filters extends React.Component {
     this.props.setViews(views)
   }
 
+  openNewFilterModal () {
+    this.setState({
+      showModal: true,
+      edit: this.generateFreshEdit()
+    })
+  }
+
   render () {
     const inputModels = sieve.makeFilterInputs(this.props.schema)
     const rules = this.props.rules || []
 
     return (
       <FilterWrapper mb={3}>
         <Flex justify='space-between'>
-          <Button primary onClick={() => this.toggleModal()}>
```

Comment:
It seems that `toggleModal()` isn't used any more, can we drop it?

Reply:
Yep can do